actool/manifest.go

```diff
@@ -245,6 +248,20 @@ func patchManifest(im *schema.ImageManifest) error {
 			app.Isolators = append(app.Isolators, *isolator)
 		}
 	}
+
+	if patchDependencies != "" {
+		dependencies := strings.Split(patchDependencies, ":")
```

Comment:
I see that the version is encoded as a label in Dependency field, we could somehow support that